### PR TITLE
Add general installation info

### PR DESCRIPTION
**Pull request by y-vectorfield**
Hello, I added general installation info into README. If you have any questions and comments, please contact me by message.

### CODE REVIEW

**Comment by y-vectorfield** [comment]
@frostming 
I submit pull request again.(#4205 )

**Comment by frostming** [comment]
@y-vectorfield You don't have to re submit a PR, you can just commit to the existing branch.

File: README.md [hunk 1]
```diff
@@ -55,6 +55,10 @@ Or, if you\'re using FreeBSD:
 
     # pkg install py36-pipenv
 
+When none of the above is an option(you need to use Python 3.6+):
```
Comment (frostming) [inline README.md:58]: What is the reason Python 3.6+ is required? Since pipenv officially supports Python 2.7, this will confuse people.

Comment (y-vectorfield) [reply README.md:58]: Oh, sorry. I'm misunderstanding about this. I modify this right now.

**Comment by y-vectorfield** [comment]
> @y-vectorfield You don't have to re submit a PR, you can just commit to the existing branch.
> 
> A PR is not a one-off task, it often needs several rounds of review and updates.

Oh, sorry. I renamed this branch's name...

**Comment by y-vectorfield** [comment]
@frostming 
Thank you very much for your approving.

**Pull request merged**